src/tcgwars/logic/impl/gen1/IntroPack.Groovy

```diff
@@ -391,7 +392,6 @@ static Type C = COLORLESS, R = FIRE, F = FIGHTING, G = GRASS, W = WATER, P = PSY
 							flip 3, {damage 10}
 						}
 					}
-				}
```

Comment:
Yeah, the entire file is suspect, actually. There's multiple cards that don't exist anywhere else.

Reply:
You can use [this Bulbapedia page](https://bulbapedia.bulbagarden.net/wiki/Intro_Pack_(TCG)) to help fix any other cards, or you can just take out the vine whip block for now since [we can confirm Meowth does not have that attack](https://bulbapedia.bulbagarden.net/wiki/Meowth_(Bulbasaur_Deck_16)). I don't think it should be left as is since you're already making changes to try and "fix" it in this spot.

(Bulbapedia has been acting up lately, if you can't access these links on the live website, try accessing them on https://web.archive.org)

Reply:
Done.